### DISCUSSION TITLE
How to enable "Now Playing" text on audio stream?

### DISCUSSION THREAD

**Discussion by z0mn** [scivision/PyLivestream#101] [Q&A]
**Describe the bug**
Hi! I've looked at `ffmpeg.py` and see a reference to drawing text of the currently playing audio file. How do I enable this? Here's the function in /usr/local/lib/python3.8/site-packages/pylivestream:

```
    def drawtext(self, text: str = None) -> List[str]:
        # fontfile=font.ttf:
        if not text:  # None or '' or [] etc.
            return []

        fontcolor = "fontcolor=white"
        fontsize = "fontsize=24"
        box = "box=1"
        boxcolor = "boxcolor=black@0.5"
        border = "boxborderw=5"
        x = "x=(w-text_w)/2"
        y = "y=(h-text_h)*3/4"

        return [
            "-vf",
            f"drawtext=text='{text}':{fontcolor}:{fontsize}:{box}:{boxcolor}:{border}:{x}:{y}",
        ]
```

**Here's an example file name:**
`1992-12-12 - <Program> - <Feature>.flac`

**System Parameters**
My standard command is as follows:
`FileGlobLivestream <absolute path to directory of audio files> youtube -glob "*.flac" -image <absolute path to image>/thumb.png -loop -shuffle
`

* OS: Ubuntu 16.04.6 LTS; Kernel: Linux 4.15.0-107-generic; Architecture: x86-64
* ffmpeg version 4.3.1

**Comment by ph33nx** [comment]
As mentioned [here](https://github.com/scivision/PyLivestream#file-livestream). If you have installed the optional [tinytag](https://pypi.org/project/tinytag/) Python module, the Title - Artist will be added automatically onto the video from the audio/video files.
